src/odemis/odemisd/mdupdater.py

```diff
@@ -341,9 +341,7 @@ def getFilterBandwidth(self, filter: Optional[model.HwComponent]) -> Union[Itera
 
         pos = filter.position.value
         bandwidth = filter.axes["band"].choices[pos["band"]]
-        if bandwidth == model.BAND_PASS_THROUGH:  # "pass-through" == no filter
-            return None
```

Comment:
You cannot just drop it. This is needed to handle correctly the case when there is both a pass-through filter and a monochromator slit, on the SPARC.